index.src.html

```diff
@@ -3672,6 +3685,50 @@
   </dd>
 </dl>
 
+Video Color Space{#video-color-space}
+-------------------------------------
+
+<xmp class='idl'>
+[Exposed=(Window,DedicatedWorker)]
+interface VideoColorSpace {
+  constructor(VideoColorSpaceInit init);
+
+  readonly attribute VideoColorPrimaries? primaries;
+  readonly attribute VideoTransferCharacteristics? transfer;
+  readonly attribute VideoMatrixCoefficients? matrix;
+  readonly attribute boolean? fullRange;
+
+  // Note: [Default] implementation would be correct.
+  VideoColorSpaceInit toJSON();
```

Comment:
Should be [Default] in the spec (that's what the note is about).

Reply:
Done.